NeosModLoader/Configuration.cs

```diff
@@ -15,7 +15,12 @@ internal static Configuration get()
             if (_configuration == null)
             {
                 // the config file can just sit next to the dll. Simple.
-                string path = $"{GetAssemblyDirectory()}\\{CONFIG_FILENAME}";
+                string path;
+                if (Environment.OSVersion.Platform==PlatformID.Unix)
+                    path = $"{GetAssemblyDirectory()}/{CONFIG_FILENAME}";
+                else
+                    path = $"{GetAssemblyDirectory()}\\{CONFIG_FILENAME}";
+

```

Comment:
If we're going to be handling cross-platform paths, we might as well do it the right way by using [Path.Combine](https://docs.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-5.0) as discussed in this [stackoverflow post](https://stackoverflow.com/questions/5758633/how-should-i-handle-windows-linux-paths-in-c-sharp)